### PR TITLE
Migrate device tracking page

### PR DESCRIPTION
**Pull request by kevcenteno**

### CODE REVIEW

**Review by wesrice — APPROVED** [review]

File: content/minfraud/evaluate-a-transaction.mdx [hunk 1]
```diff
@@ -16,10 +16,14 @@ details on our JSON API.
...
+so that the minFraud service can assign a device ID and begin collecting
+fingerprint information.  This helps detect fraudsters if they change or enable
+proxies while browsing your website.
+
+Note that, in order to be effective, the Device Tracking Add-on must, at a
+minimum, be included on the page where the IP address is captured for a
+minFraud query.
 
 Place the following code in the footer of the HTML webpage and replace
 `ACCOUNT_ID` with your MaxMind account ID:
```
Comment (rajnihatti) [inline content/minfraud/evaluate-a-transaction.mdx:29]: In the original content, `MaxMind account ID` was a link to https://www.maxmind.com/en/accounts/current/license-key. This seems useful and I think we should keep it.

Comment (rajnihatti) [reply content/minfraud/evaluate-a-transaction.mdx:29]: I just noticed this was not code you update, not sure if we should fix now or do it later

File: content/minfraud/track-devices.mdx [hunk 2]
```diff
@@ -1,6 +1,68 @@
...
+```
+
+## Cookie and web storage usage
+
+The device tracking add-on uses both a cookie and local storage as one method
+of distinguishing unique devices. The add-on sets a cookie with a two-year
+expiration for the effective second-level domain of your site; for instance, on
+`www.maxmind.com`, the cookie would be set for `maxmind.com`. Both the cookie
+and the local storage key are named `__mmapiwsid`.
+
```
Comment (rajnihatti) [inline content/minfraud/track-devices.mdx:66]: The original content also says "MaxMind may use other browser storage techniques in the future to enhance the performance of the device tracking add-on." Do we want to remove this in the new content?

File: content/minfraud/evaluate-a-transaction.mdx (hunk 1 above)
Comment (kevcenteno) [reply content/minfraud/evaluate-a-transaction.mdx]: Thanks for catching this.  I'll fix it now.

File: content/minfraud/track-devices.mdx (hunk 2 above)
Comment (kevcenteno) [reply content/minfraud/track-devices.mdx:66]: Good catch!  I've added this in the latest fix.

**Pull request merged**